### PR TITLE
remove .vscode settings

### LINKED ISSUE #49: git rm -r .vscode

### PR DESCRIPTION
**Pull request by simongdavies**
Removes .vscode directory from repo

closes #49

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -224,6 +224,10 @@ If you get the error `Error: NoHypervisorFound` and KVM or mshv is set up then t
 
 For more details on how to verify that KVM is correctly installed and permissions are correct, follow the guide [here](https://help.ubuntu.com/community/KVM/Installation).
 
+### Rust Analyzer
```
Comment (ludfjig) [inline README.md]: nit: What do you think about adding this to CONTRIBUTING.md instead?

File: src/hyperlight_host/Cargo.toml [hunk 2]
```diff
@@ -94,6 +94,7 @@ tracing-opentelemetry = "0.27.0"
 opentelemetry = "0.26.0"
 opentelemetry-otlp = { version = "0.26.0", features = ["default"] }
 opentelemetry_sdk = { version = "0.26.0", features = ["rt-tokio"] }
+opentelemetry-semantic-conventions = "0.26.0"
```
Comment (ludfjig) [inline src/hyperlight_host/Cargo.toml]: Did you intend to add this here

**Review by ludfjig — APPROVED** [review]
LGTM with minor question

File: src/hyperlight_host/Cargo.toml (hunk 2 above)
Comment (simongdavies) [reply src/hyperlight_host/Cargo.toml]: No, thanks.

File: README.md (hunk 1 above)
Comment (danbugs) [reply README.md]: ^^^ +1

**Review by danbugs — APPROVED** [review]

File: .devcontainer/Dockerfile [hunk 3]
```diff
@@ -59,5 +59,6 @@ RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y \
     && rustup target add x86_64-unknown-linux-gnu \
     && rustup target add x86_64-unknown-none      \
     && rustup target add x86_64-pc-windows-msvc   \
+    && rustup toolchain add  nightly-x86_64-unknown-linux-gnu \
```
Comment (danbugs) [inline .devcontainer/Dockerfile]:
```suggestion
    && rustup toolchain add nightly-x86_64-unknown-linux-gnu \
```

**Review by ludfjig — APPROVED** [review]

**Review by danbugs — APPROVED** [review]

**Pull request merged**